addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/FreeboxException.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.freebox.internal.api;
+
+import org.openhab.binding.freebox.internal.api.model.FreeboxResponse;
+
+/**
+ * Exception for errors when using the Freebox API
+ *
+ * @author Laurent Garnier - Initial contribution
+ */
+public class FreeboxException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    protected FreeboxResponse<?> response;
+
+    public FreeboxException(String msg) {
+        this(msg, null);
```

Comment:
Please also add a cause and forward it to parent exception

Reply:
Done